recipes/sundials/bld.bat

```diff
@@ -0,0 +1,26 @@
+mkdir build
+cd build
+
+cmake ^
+    -G "%CMAKE_GENERATOR%" ^
```

Comment:
Please change this from `"%CMAKE_GENERATOR%"` to `"NMake Makefiles"` in order to support Python 2.7 x64.